lib/src/components/NxLoadWrapper/__tests__/NxLoadWrapper.test.tsx

```diff
@@ -4,48 +4,87 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxLoadWrapper, { Props } from '../NxLoadWrapper';
-import NxLoadError from '../../NxLoadError/NxLoadError';
-import NxLoadingSpinner from '../../NxLoadingSpinner/NxLoadingSpinner';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+import NxLoadWrapper from '../NxLoadWrapper';
 
 describe('NxLoadError', function() {
   const children = <div className="children"/>,
-      retryHandler = jest.fn(),
-      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxLoadWrapper, { children, retryHandler });
+      retryHandler = jest.fn();
```

Comment:
The mock should be reset per-test.  That is, it should be instantiated either in a `beforeEach` or in each test.

See some of our other tests for how we typically define the "minimal props" that are passed into `rtlRenderElement` and `rtlRender`.  For one thing, we usually have a `minimalProps` variable rather than passing separate object literals into the two functions.  Secondly, for required function props we usually just pass a no-op function (e.g. `() => {}`) rather than a mock.  The tests that actually need a mock can set one up themselves

Reply:
> The mock should be reset per-test. That is, it should be instantiated either in a `beforeEach` or in each test.
> 
> See some of our other tests for how we typically define the "minimal props" that are passed into `rtlRenderElement` and `rtlRender`. For one thing, we usually have a `minimalProps` variable rather than passing separate object literals into the two functions. Secondly, for required function props we usually just pass a no-op function (e.g. `() => {}`) rather than a mock. The tests that actually need a mock can set one up themselves

Done 58d2866f91a819756d32c66c9066395811b84538